crates/cfg/src/lib.rs

```diff
@@ -58,6 +58,10 @@ impl CfgOptions {
         self.enabled.insert(CfgAtom::KeyValue { key, value });
     }
 
+    pub fn diff<'a>(&'a self, other: &'a CfgOptions) -> HashSet<&CfgAtom> {
+        self.enabled.difference(&other.enabled).collect()
+    }
```

Comment:
```suggestion
    pub fn difference<'a>(&'a self, other: &'a CfgOptions) -> impl Iterator<Item = &'a CfAtom> + 'a {
        self.enabled.difference(&other.enabled)
    }
```
we can use an opaque return type to save on the allocation here